python/doc/examples/meta_modeling/functional_chaos.ipynb

```diff
@@ -11,11 +11,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this basic example we are going to create a global approximation of a model response using functional chaos.\n",
+    "In this example we are going to create a global approximation of a model response using functional chaos.\n",
     "\n",
-    "Here\n",
+    "Let $h$ be the function defined by:\n",
+    "$$\n",
+    "g(\\mathbf{x}) = \\left[\\cos(x_1 + x_2), (x_2 + 1) e^{x_1}\\right]\n",
+    "$$\n",
```

Comment:
In this PR too, can you make sure you add a blank line after each closing $$ ?

Reply:
Oups. Fixed.